src/ServiceInsight.Tests/MessageBodyViewModelTests.cs

```diff
@@ -31,15 +30,14 @@ public class MessageBodyViewModelTests
         [SetUp]
         public void TestInitialize()
         {
-            eventAggregator = Substitute.For<IEventAggregator>();
             workNotifier = Substitute.For<IWorkNotifier>();
             serviceControl = Substitute.For<IServiceControl>();
             hexContent = Substitute.For<HexContentViewModel>();
             jsonContent = Substitute.For<JsonMessageViewModel>();
             xmlContent = Substitute.For<XmlMessageViewModel>();
-            selection = new MessageSelectionContext(eventAggregator);
+            selection = new MessageSelectionContext(Substitute.For<IEventAggregator>());
 
-            messageBodyFunc = () => new MessageBodyViewModel(hexContent, jsonContent, xmlContent, serviceControl, eventAggregator, workNotifier, selection);
+            messageBodyFunc = () => new MessageBodyViewModel(hexContent, jsonContent, xmlContent, serviceControl, workNotifier, selection);
```

Comment:
This change was included in the "Cleaning up usings" commit, but the related change to `MessageBodyViewModel` was not, so that commit does not compile.